app/views/unattended/provisioning_templates/provision/kickstart_default.erb

```diff
@@ -76,12 +77,17 @@ liveimg --url=<%= liveimg_url %> <%= proxy_string %>
 <%= @mediapath %><%= proxy_string %>
 <% @additional_media.each do |medium| -%>
 <% if rhel_compatible && @host.operatingsystem.name.downcase.include?("centos") && os_major >= 8 && medium[:url] && medium[:url].include?("AppStream") -%>
+<% appstream_present = true -%>
 # renamed from "<%= medium[:url] %>" for CentOS Anaconda to work
 repo --name AppStream --baseurl <%= medium[:url] %>
 <% else -%>
 repo --name <%= medium[:name] %> --baseurl <%= medium[:url] %> <%= medium[:install] ? ' --install' : '' %><%= proxy_string %>
 <% end -%>
 <% end -%>
+<% if rhel_compatible && @host.operatingsystem.name.downcase.include?("centos") && os_major >= 8 && medium_uri && medium_uri.to_s.include?("BaseOS") && !appstream_present -%>
```

Comment:
Couldn't we get rid of the extra variable and just move this condition into the `else` block that starts on line 83?

Reply:
nevermind, i just realized that it is run inside a loop (erb is a pain when we don't indent inside the tags)